### PR TITLE
0831 lwip debugging

### PR DESCRIPTION
**Pull request by pillip8282**

### CODE REVIEW

File: os/net/netmgr/netdev_lwip.c [hunk 1]
```diff
@@ -372,9 +372,15 @@ static int lwip_input(struct netdev *dev, void *frame_ptr, uint16_t len)
 	LWIP_DEBUGF(NETIF_DEBUG, ("processing pbufs\n"));
 
 	/* We iterate over the pbuf chain until we have read the entire packet into the pbuf. */
+	uint32_t total = 0;
+	uint32_t remain = len;
 	for (q = p; q != NULL; q = q->next) {
-		memcpy(q->payload, frame_ptr, q->len);
-		frame_ptr += q->len;
+		int buf_size = q->len > remain ? remain : q->len;
```
Comment (j11591) [inline os/net/netmgr/netdev_lwip.c:378]: In pbuf_ alloc, there is
>p->len = LWIP_MIN(length, PBUF_POOL_BUFSIZE_ALIGNED - LWIP_MEM_ALIGN_SIZE(offset));
In my opinion. q->len is always not not more than remain.

Comment (pillip8282) [reply os/net/netmgr/netdev_lwip.c:378]: ok. it seems unnecessary code. I'll not apply it to public.

File: os/net/lwip/src/core/mem.c [hunk 2]
```diff
@@ -409,6 +409,7 @@ void mem_init(void)
 	ram_end->next = MEM_SIZE_ALIGNED;
 	ram_end->prev = MEM_SIZE_ALIGNED;
 
+	lldbg("[pkbuild] lwip mem range %p %p \n", ram, ram_end);
```
Comment (amogh-samsung) [inline os/net/lwip/src/core/mem.c:412]: could you confirm what "pkbuild" is please?
Seems to be a personal log, in that case could you generalize or remove it please?

File: os/net/netmgr/netdev_lwip.c [hunk 3]
```diff
@@ -372,9 +372,15 @@ static int lwip_input(struct netdev *dev, void *frame_ptr, uint16_t len)
...
+		lldbg("[PK] %p %p %p %p %d %d %d %d\n", p, q, q->payload, frame_ptr, len, total, q->len, remain);
```
Comment (amogh-samsung) [inline os/net/netmgr/netdev_lwip.c:379]: Same as previous, could you confirm the significance of "PK" here please?

**Comment by pillip8282** [comment]
Issue is resolved by https://github.com/Samsung/TizenRT/pull/5203. So close this test PR

**Pull request closed without merging**